src/ru/skypro/Employee.java

```diff
@@ -0,0 +1,57 @@
+package ru.skypro;
+
+public class Employee {
+    private static int counter;
+    private final int id;
+    private String name;
+    private String patronymic;
+    private String lastName;
+    private int unit;
+    private double salary;
+
+    public Employee(String name, String patronymic, String lastName, int unit, double salary) {
+        this.id = counter;
```

Comment:
можно объединить с инкрементом, в идеале лучше начинать id с 1, чем с нуля, чем нам поможет префиксный инкремент (сначала увеличит значение, потом его вернет)
```suggestion
        this.id = ++counter;
```